src/clj/kixi/hecuba/api/datasets.clj

```diff
@@ -67,7 +67,9 @@
       :subtract parent-period
       :divide "INSTANT"
       :multiply-series-by-field parent-period
-      :divide-series-by-field parent-period)))
+      :divide-series-by-field parent-period
+      :total-usage-weekly "PULSE"
+      :total-usage-monthly "PULSE")))
```

Comment:
None - the card I'm currently working on is to add total usage calculation. The bottom two lines are adding those (weekly and monthly)